packages/lesswrong/server/apolloServer.ts

```diff
@@ -227,6 +228,9 @@ export function startWebserver() {
     const user = await getUserFromReq(request);
     const themeOptions = getThemeOptions(request, user);
     const stylesheet = getMergedStylesheet(themeOptions);
+    const externalStylesPreload = globalExternalStylesheets.map(url =>
+      `<link rel="stylesheet" type="text/css" href="${url}">`
```

Comment:
`rel="preload"` is used when there's going to be another inclusion of the same stylesheet again, later in the page; it changes the semantics so that it doesn't actually apply the stylesheet to the page, it only downloads it so that it can be applied later. In this case we're only specifying the stylesheet once, unconditionally, so we don't do that.